src/modules/fancyzones/editor/FancyZonesEditor/EditorOverlay.xaml.cs

```diff
@@ -14,14 +14,13 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class EditorOverlay : Window
     {
-        public static EditorOverlay Current { get; set; }
 
-        private readonly Settings _settings = ((App)Application.Current).ZoneSettings[0];
+        public Settings Settings = App.ZoneSettings[MonitorVM.CurrentMonitor];

```

Comment:
Are they being accessed outside this class? I don't see them referenced anywhere else

Reply:
You are right! This was a legacy change that I forgot to re-edit ~